pkg/common/utils/gang_utils.go

```diff
@@ -123,24 +125,42 @@ func GetTaskGroupsFromAnnotation(pod *v1.Pod) ([]v1alpha1.TaskGroup, error) {
 	return taskGroups, nil
 }
 
-func GetPlaceholderTimeoutParam(pod *v1.Pod) (int64, error) {
+func GetSchedulingPolicyParam(pod *v1.Pod) (int64, error, string, error) {
```

Comment:
Please use two methods instead of mixing the 2 calls. You can export the common parts into one method. See my comment in the general.go file